core/src/main/java/org/eclipse/krazo/security/CsrfValidateFilter.java

```diff
@@ -105,12 +110,14 @@ public void filter(ContainerRequestContext context) throws IOException {
 
             // Validate CSRF
             final Form form = formEntityProvider.getForm(context);
-            final List<String> tokenValues = form.asMap().get(token.getParamName());
+            final List<String> tokenValues = form.asMap()
+                .get(token.getParamName());
             if (tokenValues == null || tokenValues.isEmpty()) {
                 throw new CsrfValidationException(messages.get("CsrfFailed", "missing field"));
             }
 
-            if (!token.getValue().equals(tokenValues.get(0))) {
+            if (!token.getValue()
+                .equals(tokenValues.get(0))) {
```

Comment:
There are some weird code format changes included in the PR. Could you perhaps keep the original formatting?

Reply:
Yes I can revert it and keep an eye on it in the next PRs. Do we have somewhere a common formatter for Krazo?

Reply:
AFAIK there is no code format profile yet. AFAIK the original code used the IntelliJ IDEA default configuration. But I can check that. I'll create an issue for tracking this. Thanks for bringing this up.